lib/api/objectPutPart.js

```diff
@@ -218,6 +219,10 @@ export default function objectPutPart(authInfo, request, streamingV4Params, log,
             });
             return cb(err);
         }
+        pushMetric('uploadPart', log, {
+            bucket: bucketName,
+            newByteLength: size,
```

Comment:
Metrics for overwrites are currently not handled in this API, but is being addressed by PR https://github.com/scality/S3/pull/472. Accordingly, the Utapi branch this PR is dependent on is not enforcing that property for this metric. If https://github.com/scality/S3/pull/472 is merged first before this one, then I will update this.